src/aci/aci.cc

```diff
@@ -2153,7 +2112,7 @@ Timer build_space;
 outfile->Printf("\n  Time spent building the model space: %1.6f", build_space.get());
         // Check if P+Q space is spin complete
         if (spin_complete_) {
-            PQ_space.make_spin_complete(ncmo_); // <- xsize
+            PQ_space.make_spin_complete(nact_); // <- xsize
```

Comment:
Yes, the number of determinants added in this step was extremely large, and the Davison-Liu solver could not find a solution with the correct spin. Upon printing the determinants, many of the added determinants had a lot of electrons added, probably because the occupations beyond nact_ but within ncmo_ were not defined.

Reply:
Ok, thanks!